### PR TITLE
September newsletter

### PR DESCRIPTION
**Pull request by milancurcic**
Please review and add anything that I missed. As usual, add yourself to the list of authors if you add, edit, or even just review content.

I left specific TODOs for @certik for LFortran news and @gklimowicz for Flang news. 

Sorry for the short notice! We'll try to start this earlier in the month. I got carried away and time flew by. If possible, aiming to merge on Tuesday, September 1.

### CODE REVIEW

File: _posts/2020-09-01-Fortran-Newsletter-September-2020.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+TODO @gklimowicz
+
+### LFortran
+
+TODO @certik
+
+## Events
+
+* We had our fourth Fortran Monthly call on August 20.
+You can read watch the recording below:
```
Comment (everythingfunctional) [inline _posts/2020-09-01-Fortran-Newsletter-September-2020.md:88]:
```suggestion
You can watch the recording below:
```

**Review by jvdp1 — APPROVED** [review]
It looks good to me. Thank you.
I hope I can find some times next month to be more active.

**Comment by certik** [comment]
Thanks! I have updates for LFortran and also for GFortran from Thomas König. I'll add it in later today.

**Comment by milancurcic** [comment]
#build_preview

**Comment by milancurcic** [comment]
The contributions are not displaying for me in the preview. @certik and @LKedward, is perhaps the contribution data that needs to be updated? There is a script that does it? I forget how it works.

**Comment by LKedward** [comment]
> The contributions are not displaying for me in the preview. @certik and @LKedward, is perhaps the contribution data that needs to be updated? There is a script that does it? I forget how it works.

I'll run it now and push the updated json files.

**Comment by LKedward** [comment]
#build_preview

**Comment by milancurcic** [comment]
Thank you all and thank you Laurence for updating the contributions data. I think it's good to go. I will merge in one hour.

**Comment by awvwgk** [comment]
#delete_preview

**Pull request merged**